n_t/msmc.py

```diff
@@ -30,10 +30,12 @@ def write_msmc_file(path, num_sampled_genomes_msmc):
     """
     for sample_size in num_sampled_genomes_msmc:
         ts = prune_tree_sequence(path, sample_size)
-        sep = path.split(".")
-        chrom = sep[1]
-        sep.insert(1,str(sample_size))
-        output = ".".join(sep) + ".multihep.txt"
+        dirr = os.path.dirname(path)
```

Comment:
Can do this more cleanly using pathlib.

Reply:
@jeromekelleher should I merge this and we can change later?

Reply:
can change later, totally not a blocker.